meson.build

```diff
@@ -3,13 +3,17 @@ project('ml-api', 'c', 'cpp',
   license: ['Apache-2.0'],
   meson_version: '>=0.50.0',
   default_options: [
+    'b_asneeded=false',
     'werror=true',
     'warning_level=1',
     'c_std=gnu89',
     'cpp_std=c++14'
   ]
 )
 
+add_global_link_arguments('-Wl,--no-as-needed', language: 'c')
+add_global_link_arguments('-Wl,--no-as-needed', language: 'cpp')
```

Comment:
`--as-needed` is set as default in Tizen build infra to reduce the number of symbols in the dynamic section.
However, in this case, it is reasonable for me.